src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -65,6 +66,12 @@ internal partial class Binder
             RoslynDebug.Assert((object)destination != null);
             RoslynDebug.Assert(!isCast || conversionGroupOpt != null);
 
+            if (source is BoundDelegateCreationExpression delegateCreationExpression &&
+                delegateCreationExpression.MethodOpt is not null)
+            {
+                ReportDiagnosticsIfObsolete(diagnostics, delegateCreationExpression.MethodOpt, delegateCreationExpression.Argument.Syntax, hasBaseReceiver: false);
+            }
```

Comment:
@333fred I'm unsure of this is how you'd prefer to fix that. Can you take a look?

Reply:
I believe it would be better to put this check in `MethodGroupConversionHasErrors`, like the change to report UnmanagedCallersOnly errors.

---
In reply to: [490915535](https://github.com/dotnet/roslyn/pull/47332#discussion_r490915535) [](ancestors = 490915535)